js/controller.js

```diff
@@ -1451,13 +1451,14 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     onChangeClosedCaptionLanguage: function(event, language) {
+      var noneLanaguage = 'none';
```

Comment:
Can this value be defined as a constant in the skin's constants.js? Also, 'lanaguage' is misspelled.

Reply:
sure will do

Reply:
@jpfranco I have added the constant to the constants.js. Please check it out.

Reply:
@jpfranco Please approve the PR.